src/nyaml/cli.py

```diff
@@ -74,6 +74,8 @@ def split_name_and_extension(file_path):
     """
     path = Path(file_path)
     ext = "".join(path.suffixes)
+    # assuming that in NeXus yaml and nxdl files follow the pattern
```

Comment:
Is this assumption also mentioned in the readme and documentation?

Reply:
@mkuehbach add to separate doc editing PR